.circleci/config.yml

```
@@ -79,6 +81,33 @@ commands:
           name: Run pylint
           command: |
             poetry run pylint --rcfile=scripts/.pylintrc --disable=R --ignore=requirements.txt scripts/*
+  deploy:
+    description: "Upload package to PyPI"
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: poetry install --no-dev
+      - run:
+          name: Verify tag version
+          command: poetry run python -m gvmtools.version verify ${CIRCLE_TAG}
+      - run:
+          name: Install twine
+          command: poetry run pip install twine
+      - run:
+          name: Initialize .pypirc
+          command: |
+            echo -e "[distutils]" >> ~/.pypirc
```

```suggestion
            echo -e "[distutils]" > ~/.pypirc
```

This would ensure the `.pypirc` is always flushed before initialization, even if it should already exist for whatever reason.